setup.py

```diff
@@ -44,7 +44,7 @@ def read_utf8(filename):
     # For a list of valid classifiers, see
     # https://pypi.python.org/pypi?%3Aaction=list_classifiers
     classifiers=[
-        'Development Status :: 3 - Alpha',
```

Comment:
I think you are looking for `Development Status :: 4 - Beta`, vide list of valid classifiers: https://pypi.python.org/pypi?%3Aaction=list_classifiers

Reply:
My bad, thanks!